.travis.yml

```diff
@@ -30,6 +32,12 @@ matrix:
   - env: NODE_NVM_VERSION="0.12.18"
 before_install:
 - if [[ $TRAVIS_OS_NAME == "linux" ]]; then export CXX=g++-4.8; fi
+- if [[ $ELECTRON_VERSION ]]; then
```

Comment:
via https://electronjs.org/docs/tutorial/using-native-node-modules#using-npm

Reply:
lets try this

Reply:
Thank you, it now works when using `npm rebuild` (`npm rebuild --runtime=electron --target=2.0.7 --dist-url=https://atom.io/download/electron`):

```
node-pre-gyp http GET https://hummus.s3-us-west-2.amazonaws.com/hummus/v1.0.88/electron-v2.0-darwin-x64.tar.gz
node-pre-gyp http 200 https://hummus.s3-us-west-2.amazonaws.com/hummus/v1.0.88/electron-v2.0-darwin-x64.tar.gz
node-pre-gyp info install unpacking binding/hummus.node
node-pre-gyp info tarball done parsing tarball
[hummus] Success: "/Users/houssam/method-writer/app/node_modules/hummus/binding/hummus.node" is installed via remote
```

I'm not sure if it's worth it, but many people use `electron-rebuild` which leverages [prebuild](https://www.npmjs.com/package/prebuild) instead of `node-pre-gyp`. It's not that big of a deal though.